multires-lyrics-search/flows/query.yml

```diff
@@ -16,7 +16,7 @@ pods:
     shards: $JINA_SHARDS
     separated_workspace: true
     polling: all
-    uses_reducing: _merge_all
+    uses_after: pods/chunk_merger.yml
```

Comment:
U can use _reduce_all I think or _reduce I do not remember exactly

Reply:
That didn't work. It's because it need to be set to `traversal: ['cm']`.